lib/streamlit/elements/checkbox.py

```diff
@@ -35,6 +35,8 @@ def checkbox(self, label, value=False, key=None):
             If this is omitted, a key will be generated for the widget
             based on its content. Multiple widgets of the same type may
             not share the same key.
+        on_change : callable
```

Comment:
Is anything passed to the callback? I think we should include such information (including if it shouldn't have any arguments)